### PR TITLE
Add wrappers for `gemv_batched!`

### PR DESCRIPTION
**Pull request by lpawela**
This adds wrappers and updates `@ccall` arguments for various versions of batched matrix vector multiplication.

### CODE REVIEW

File: lib/cublas/wrappers.jl [hunk 1]
```diff
@@ -323,17 +323,17 @@ for (fname, elty) in ((:cublasDgemv_v2,:Float64),
                        A::StridedCuMatrix{$elty},
                        X::StridedCuVector{$elty},
```
Comment (maleadt) [inline lib/cublas/wrappers.jl]: If you want to change capitalization for consistency, also change `X` to `x` then.

File: lib/cublas/libcublas.jl [hunk 2]
```diff
@@ -3355,11 +3355,11 @@ end
     initialize_context()
     @ccall libcublas.cublasSgemvStridedBatched(handle::cublasHandle_t,
                                                trans::cublasOperation_t, m::Cint, n::Cint,
-                                               alpha::Ptr{Cfloat}, A::Ptr{Cfloat},
```
Comment (maleadt) [inline lib/cublas/libcublas.jl:3358]: These wrappers are auto-generated, so you should also update https://github.com/JuliaGPU/CUDA.jl/blob/master/res/wrap/cublas.toml

Comment (lpawela) [reply lib/cublas/libcublas.jl:3358]: I can't find these wrappers in that file. They only appear in this file. Probably that is why there was some type inconsistency.

Comment (maleadt) [reply lib/cublas/libcublas.jl:3358]: Yes, you need to add an entry. The purpose is that file is to add additional rewrites to the autogenerated headers.

Comment (lpawela) [reply lib/cublas/libcublas.jl:3358]: Ok. As I understand the structure, I only declare which variable's types I want to rewrite. Two questions:
1. Variable numbering begins from `1`?
2. What is the "default" type of a variable?

Comment (maleadt) [reply lib/cublas/libcublas.jl:3358]: > Variable numbering begins from `1`?

Yes



> What is the "default" type of a variable?

What's currently being generated based on the C headers. The TOML only provides overrides.

**Comment by maleadt** [comment]
Looks like `cublasSgemvBatched` is only available on CUDA 11.6+, so you'll have to gate the tests accordingly. Maybe gate on the CUBLAS version; CUDA 11.6 ships CUBLAS 11.9.2 so `if CUBLAS.version() >= v"11.9"` or so.

**Comment by maleadt** [comment]
Thanks!

**Pull request merged**